validator/validator-main.protoascii

```diff
@@ -2735,6 +2735,18 @@ tags: {
   mandatory_ancestor: "FORM [method=POST]"
   spec_url: "https://amp.dev/documentation/components/amp-form/"
 }
+tags: {
+  html_format: AMP
+  tag_name: "INPUT"
+  spec_name: "INPUT [form]"
```

Comment:
Please add a `descriptive_name` as well. This is what would be used in the error messages shown to developers.

https://github.com/ampproject/amphtml/blob/master/validator/validator.proto#L569

Reply:
Resolving as I believe there shouldn't be a new tagspec for this but included in input-common-attr attribute list.